integration-test/example.js

```diff
@@ -0,0 +1,11 @@
+const got = require('../dist/source');
+
+(async () => {
+	try {
+		await got('https://google.com');
```

Comment:
Can you create a local server instead? We should avoid querying 3rd party servers.

Reply:
Actually for an integration test... I don't know.

Reply:
I think these tests should be as close as possible to real usages.

Reply:
Can we use 3rd-party packages for this? They use Got more extensively I think.

Reply:
Like https://github.com/yarnpkg/berry or https://github.com/renovatebot/renovate or https://github.com/MONEI/Shopify-api-node/ or (it goes on and on).